### DISCUSSION TITLE
Gitkraken

### DISCUSSION THREAD

**Discussion by yuri-becker** [catppuccin/catppuccin#2216] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

[As of version 8.2 Gitkraken supports custom themes](https://www.gitkraken.com/blog/gitkraken-client-v8-2#gitkraken-themes).

Should be fairly easy to implement, sadly there is no official distribution method for themes.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by nekowinston** [comment]
We have an open issue for that in https://github.com/catppuccin/catppuccin/issues/2170, I'd appreciate if you could check it out and leave comments there, if anything stands out to you in the current port request. :slightly_smiling_face:
